src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -62,45 +64,45 @@ const PurchaseOrderListTable = ({
       label: 'react.purchaseOrder.viewOrderDetails.label',
       defaultLabel: 'View order details',
       leftIcon: <RiInformationLine />,
-      href: '/openboxes/order/show',
+      href: ORDER_URL.show,
```

Comment:
I think this might not work due to the same reason as for invoice actions I described in details above.